### PR TITLE
apply batch of changes

### LINKED ISSUE #3: feat: tab selector decoration
Add the little circle below the current selected tab (about, create treasury and disclaimer)

### LINKED ISSUE #6: feat: current page indicator
- When page is `what-are-the-risks`, `what-are-yearn-vaults`, `what-is-cowswap` or `what-is-mev`, the `disclaimer` nav tag should be decorated (black color and little dot) and the two other should be reset (grey color, no dot)

- When page is `/`, `learn-more` or `how-it-works` the `about` nav tag should be decorated (black color and little dot) and the two other should be reset (grey color, no dot)

- On other pages, the `create treasury` nav tag should be decorated (black color and little dot) and the two other should be reset (grey color, no dot)

### LINKED ISSUE #14: feat: input component

### LINKED ISSUE #16: feat: withdraw, "final final", "done!", and treasury pages
withdraw: 
image

### LINKED ISSUE #17: fix: box width should be fixed across all "create-treasury" flow
fix width on "keep-eth"

### PR DESCRIPTION
**Pull request by MarcoWorms**
closes #17 
closes #16
closes #6 
closes #3 
closes #14

### CODE REVIEW

File: components/Header.tsx [hunk 1]
```diff
@@ -1,21 +1,46 @@
...
 	return (
 		<header className={'flex flex-row justify-between items-center py-10'}>
-			<Link href={'/'}>
-				<div className={'cursor-pointer'}>
-					<LogoNFTreasury />
-				</div>
-			</Link>
+			<div>
+				<LogoNFTreasury />
+			</div>
```
Comment (Majorfi) [inline components/Header.tsx]: Let's keep the home redirect here :)
```suggestion
			<Link href={'/'}>
				<div className={'cursor-pointer'}>
					<LogoNFTreasury />
				</div>
			</Link>
```

File: components/Header.tsx [hunk 2]
```diff
@@ -1,21 +1,46 @@
...
-					<p className={'cursor-pointer text-neutral-700'}>{'about'}</p>
+					<p className={`cursor-pointer ${isAboutPage ? 'text-primary-100 dot-below' : 'text-neutral-400' }`}>{'about'}</p>
 				</Link>
-				<Link href={'/create-treasury'}>
-					<p className={'cursor-pointer text-neutral-500'}>{'create treasury'}</p>
+				<Link href={'/connect-wallet'}>
+					<p className={`cursor-pointer  ${isCreateTreasuryPage || isPortfolioPage ? 'text-primary-100 dot-below' : 'text-neutral-400' }`}>
+						{isPortfolioPage ? 'portfolio' : 'create treasury'}
+					</p>
 				</Link>
```
Comment (Majorfi) [inline components/Header.tsx]: This tag text should stay `create treasury`, but the redirect link should change instead :)

File: pages/connect-wallet.tsx [hunk 3]
```diff
@@ -5,9 +5,9 @@ import	WithShadow				from	'components/WithShadow';
 
 function	DisclaimerPage(): ReactElement {
 	return (
-		<div className={'flex items-center w-6/12 h-full'}>
+		<div className={'flex items-center h-full'}>
 			<WithShadow role={'large'}>
-				<Card className={'flex flex-col justify-between w-full h-[500px]'}>
+				<Card className={'flex flex-col justify-between w-[600px] h-[600px]'}>
```
Comment (Majorfi) [inline pages/connect-wallet.tsx:10]: Maybe could be nice to put this class in the css file to make it more generic

File: pages/connect-wallet.tsx [hunk 4]
```diff
@@ -22,15 +22,15 @@ function	DisclaimerPage(): ReactElement {
...
+								<Link href={'/keep-eth'}>
 									<Button className={'w-[176px]'}>
 										{'Click'}
 									</Button>
-								</WithShadow>
-							</div>
-						</Link>
+								</Link>
+							</WithShadow>
+						</div>
```
Comment (Majorfi) [inline pages/connect-wallet.tsx]: Merge conflict error, you should keep it as it was
```
<Link href={'/keep-eth'}>
	<div>
		<WithShadow role={'button'}>
			<Button className={'w-[176px]'}>
				{'Click'}
			</Button>
		</WithShadow>
	</div>
</Link>
```

**Review by Majorfi — CHANGES_REQUESTED** [review]
Overall good job ser!
A few elements for a RFC, nothing much for now!

**Comment by Majorfi** [comment]
@MarcoWorms please check #19 before merging!

File: components/Header.tsx (hunk 2 above)
Comment (MarcoWorms) [reply components/Header.tsx]: on the "treasury" page it shows "portfolio" in figma!

Comment (MarcoWorms) [reply components/Header.tsx]: image

Comment (Majorfi) [reply components/Header.tsx]: Good catch
Let's replace it anyway until we got the implementation \o

**Pull request merged**